scripts/run-multichain.sh

```diff
@@ -1,29 +1,17 @@
 #!/usr/bin/env bash
 
-killall sifnoded sifnodecli
+killall sifnoded 
+killall hermes 
 
 #sifnodecli rest-server &
-#sifnoded start
 
-sifnoded start --home ~/.sifnode-2 --p2p.laddr 0.0.0.0:27656  --grpc.address 0.0.0.0:9091 --address tcp://0.0.0.0:27660 --rpc.laddr tcp://127.0.0.1:27658 >> abci_2.log 2>&1  &
-sifnoded start --home ~/.sifnode-1 --p2p.laddr 0.0.0.0:27655  --grpc.address 0.0.0.0:9090 --address tcp://0.0.0.0:27659 --rpc.laddr tcp://127.0.0.1:27657 >> abci_1.log 2>&1  &
-rm -rf ~/.ibc-setup/last-queried-heights.json
-#Reset connections
-ibc-setup ics20 --mnemonic "race draft rival universe maid cheese steel logic crowd fork comic easy truth drift tomorrow eye buddy head time cash swing swift midnight borrow" --home ~/.ibc-122
-ibc-setup ics20 --mnemonic "race draft rival universe maid cheese steel logic crowd fork comic easy truth drift tomorrow eye buddy head time cash swing swift midnight borrow" --home ~/.ibc-31
-ibc-relayer start -i -v --poll 10 --home ~/.ibc-122
-ibc-relayer start -i -v --poll 10 --home ~/.ibc-23
-ibc-relayer start -i -v --poll 10 --home ~/.ibc-31
-#Created channel:
-#  localnet-1: transfer/channel-0 (connection-0)
-#  localnet-2: transfer/channel-0 (connection-0)
+echo "starting sifnode servers"
```

Comment:
Do you think it would make sense to add a gaia network here as well ? The steps can be pretty much identical , and it would help us test more scenarios

Reply:
I guess I'd have to say that, in the interest of time, less is more.